elasticsearch.md

```diff
@@ -920,6 +920,46 @@ After adding the `filter` rule \(using the block duplication strategy\).
       kibana_index: ".kibana_@{user}"
 ```
 
+#### `response_fields`
+
+This rule allows filtering Elasticsearch responses using defined list of fields. It works in very similar way to fields rule. However it **doesn't make use of Field Level Security \(FLS\)** and could be applied to every response returned by Elasticsearch.
+
+It could be configured in two modes: *whitelist* passing only defined fields or *blacklist* filtering out defined fields.
+
+**Blacklist mode**
+
+Specifies which fields should be filtered out with `~` (other fields from mapping become allowed implicitly). Example:
+
+`fields: ["~excluded_fields_prefix_*", "~excluded_field", "~another_excluded_field.nested_field"]`
+
+Return documents but deprived of the fields that:
+  * start with `excluded_fields_prefix_`
+  * are equal to `excluded_field`
+  * are equal to `another_excluded_field.nested_field`
+
+**Whitelist mode**
+
+In this mode rule is configured to filter out each field that isn't defined in rule.
+
+`response_fields: ["allowed_fields_prefix_*", "_*", "allowed_field.nested_field.text"]`
+
+Return documents deprived of all the fields, except the ones that:
+ * start with `allowed_fields_prefix_`
+ * start with underscore
+ * are equal to `allowed_field.nested_field.text`
+
+**NB:** You can only provide a full black list or white list. Grey lists \(i.e. `["~a", "b"]`\) are invalid settings and ROR will refuse to boot up if this condition is detected.
+
+*Example*: allow only `cluster_name` and `status` field in cluster health response:
+
+```
+- name: "Filter cluster health response"
+  uri_re: "^/_cluster/health"
+  response_fields: ["cluster_name", "status"]
+```
+
+Any field could be filtered using this rule.
```

Comment:
maybe we should show an example of full `/_cluster/health`response body and the filtered one?